projects/ad713x_fmcz/src/parameters.h

```diff
@@ -78,10 +78,6 @@
 #define AD7134_FMC_CH_NO		8
 #define AD7134_FMC_SAMPLE_NO		1024
 
-#define SPI_ENGINE_RX_ADDR		0x800000
```

Comment:
it looks like these defines are still used here :
https://github.com/analogdevicesinc/no-OS/blob/bfb5fecff0b18a7494bd0db0e7fd77467d8eae5e/projects/ad713x_fmcz/src/ad713x_fmc.c#L300-L302